### ISSUE TITLE
Subscribe to list via checkout doesn't work anymore

### ISSUE THREAD

**Issue by flowdee** [fdmedia-io/edd-sendy#6]
```
Hi,
I am unable to get the Sendy EDD extension to work on my site.
Sendy itself works on my site, this is the same list I am trying to subscribe customers to after purchase:
https://www.shapingwaves.com/newsletter/

However, none of the EDD integration works.
I tried with free products, paid products, entered the list manually per product while keeping it entered in the global settings, tried with "show checkbox" enabled and disabled. Nothing works.

I am not using any caching, my server has all the PHP, Wordpress and MYSQL listed to be needed for the plugin.

I am attaching my info in some screenshots as well as the EDD sysinfo file, do you have any idea why this is not working?
```

**Comment by flowdee** [comment]
https://secure.helpscout.net/conversation/1201355562/26090?folderId=407335

**Comment by flowdee** [comment]
https://secure.helpscout.net/conversation/1196104124/25926?folderId=407334

**Comment by webdj-agency** [comment]
Fixed

**Comment by flowdee** [comment]
Released with v1.1
